spec/spec_helper_acceptance_local.rb

```diff
@@ -10,5 +10,14 @@
   c.before :suite do
     # Download the plugins to ensure up-to-date facts
     PuppetLitmus::PuppetHelpers.run_shell('/opt/puppetlabs/bin/puppet plugin download')
+    pp = <<-PUPPETCODE
+    class {'puppet_enterprise':
+     enable_system_metrics_collection => true,
+     }
+    package{'sysstat':
+     ensure   => installed,
+     }
+    PUPPETCODE
+    apply_manifest(pp)
```

Comment:
after the apply manifest i would add the hiera to the common.yaml, restarting the puppet service will both trigger puppet to apply the hiera data, and unlike simply running puppet, will ensure there are no clashing runs

run_shell('echo "puppet_enterprise::enable_system_metrics_collection: true" >> /etc/puppetlabs/code/environments/production/data/common.yaml') 
run_shell('puppet resource service puppet ensure=stopped')
run_shell('puppet resource service puppet ensure=running')